src/common/com/intellij/plugins/haxe/model/HaxeClassModel.java

```diff
@@ -356,7 +597,28 @@ public void addField(String name, SpecificTypeReference type) {
     this.getDocument().addTextAfterElement(getBodyPsi(), "\npublic var " + name + ":" + type.toStringWithoutConstant() + ";\n");
   }
 
-  public void addMethod(String name) {
-    this.getDocument().addTextAfterElement(getBodyPsi(), "\npublic function " + name + "() {\n}\n");
+  public void addMethods(HaxeMethodBuilder... methodBuilders) {
+    String out = "";
+    for (HaxeMethodBuilder builder : methodBuilders) {
+      out += "public " + builder.toString() + "\n";
```

Comment:
Why always add public?  Shouldn't we match the accessibility of the underlying methods?  And, if this is an interface,  "public" is redundant.